src/main/java/com/powsybl/openloadflow/network/LfBus.java

```diff
@@ -135,7 +135,7 @@ default double getHighVoltageLimit() {
 
     Optional<LfShunt> getSvcShunt();
 
-    Optional<LfLoad> getLoad();
+    List<LfLoad> getLoads();
```

Comment:
Why?

Reply:
Because we have to group IIDM load with same load model (so same values of parameters c0, c1, c2 of a zip load for instance), and we can result to several LF loads